discord/http.py

```diff
@@ -523,22 +522,23 @@ def send_multipart_helper(
 
         attachments = []
         form.append({"name": "payload_json"})
-        for index, file in enumerate(files):
-            attachments.append(
-                {
-                    "id": index,
-                    "filename": file.filename,
-                    "description": file.description,
-                }
-            )
-            form.append(
-                {
-                    "name": f"files[{index}]",
-                    "value": file.fp,
-                    "filename": file.filename,
-                    "content_type": "application/octet-stream",
-                }
-            )
+        if files:
```

Comment:
Files is already required
https://github.com/Pycord-Development/pycord/blob/8e941f31e276eb5c46a0df4e41ab512e59023002/discord/http.py#L493

Reply:
Good catch, not sure how I missed that lol